mopidy/http/ext.conf

```diff
@@ -4,3 +4,4 @@ hostname = 127.0.0.1
 port = 6680
 static_dir =
 zeroconf = Mopidy HTTP server on $hostname
+allowed_origins =
```

Comment:
Does `localhost:6600` work without whitelisting it, or should we make that the default?